src/components/App.js

```diff
@@ -1,31 +1,37 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { Router, IndexRoute, Route, browserHistory, Link } from 'react-router';
-import { fetchWeatherByZip } from '../actions/index';
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import { fetchLocalWeather } from '../actions/index';
+// import { Router, IndexRoute, Route, browserHistory, Link } from 'react-router';
+// import { fetchWeatherByZip } from '../actions/index';
+
+import localWeather from '../actions/data/weather';
 
 import Header from './Header';
```

Comment:
First hot take as I dig in a little more. You're requiring `components/Header.js`, but based on your question it sounds like you _want_ `containers/Header.js`—as that's the one you connected to Redux. My suspicion is that if you do the following, then you should be alright.

```js
import Header from '../containers/Header';
```